src/pages/Index/index.tsx

```diff
@@ -72,6 +72,7 @@ const UserInfoAvatarButton: Component = () => {//头像组件
     return (
         <>
             <IconButton
+                id="user-info-avatar-button"
```

Comment:
I can accept using id for testing, but it's better to use `aria-` tags to identify the elements.

I found a related issue from testing-library/react-testing-library: https://github.com/testing-library/react-testing-library/issues/16.